doc/source/intro_introduction.rst

```diff
@@ -0,0 +1,88 @@
+.. _introduction:
+
+************
+Introduction
+************
+
+The `Data Working Group <http://ga4gh.org/#/>`_ of the
+`Global Alliance for Genomics and Health <http://genomicsandhealth.org/>`_
+developed this
+`web API <http://ga4gh.org/documentation/api/v0.5.1/ga4gh_api.html#/>`_
+to facilitate access to and exchange of genomics data across remote sites. 
+
+--------------------------
+Why use this web API?
+--------------------------
+
+This API is specifically designed to allow sharing of genomics data without having to exchange complete experiments.
+With this API, you can
+
+* Exchange genome annotations, DNA sequence reads, reference-based alignments, metadata, and variant calls
+* Request alignments and variant calls for one genome or a million.
+* Explore data by slicing alignments and variants by genomic range across one or multiple samples
+* Interactively process entire cohorts
+
+--------------------------
+If you need background
+--------------------------
+For more details on web APIs, see `this wikipedia page <https://en.wikipedia.org/wiki/Web_API>`_  
+
+ReST protocols for data transfer are described `here <https://en.wikipedia.org/wiki/Representational_state_transfer>`_
+
+--------------------------
+The GA4GH web API
+--------------------------
+This API was created to enable researchers to better access and exchange genomic data across remote sites. Instead of downloading complete BAM files or
+whole genome annotations, the API allows retrieval of information on, for instance, single genes or genomic regions.
+
+For a full list of the GA4GA API goals, see :ref:`apigoals`
```

Comment:
@dglazer Pleaase feel free to edit away and commit to the topic branch.

Thanks!!!